src/Pages/Properties/Status.php

```diff
@@ -18,10 +18,17 @@ class Status implements PropertyInterface
 {
     private function __construct(
         private readonly PropertyMetadata $metadata,
-        public readonly StatusOption $option,
+        public readonly StatusOption|null $option,
```

Comment:
I do not think that a status can ever be null. The SDK creates always valid objects. Creating an empty status property and sending it to the Notion API would result in an error.

Reply:
Also, please run the Psalm tests before pushing. 

```bash
composer run test:psalm
```

Reply:
I made the change and run the test locally but didn't find a way to "run the tests before pushing". 

How can I do it? i'm still a newbie on PR, the one you validated was actually my [first PR ever](https://firstpr.me/#ren0v) ;)